src/net/wooga/jenkins/pipeline/BuildUtils.groovy

```diff
@@ -1,6 +1,6 @@
 package net.wooga.jenkins.pipeline
 
-import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
+import com.cloudbees.groovy.cps.NonCPS
```

Comment:
These are the changes I meant. Are these needed for this PR? If yes all is fine.

Reply:
The imports changes are needed as these imports were not working and not being used anymore. The `@NonCPS` annotation import is needed in order for it to compile. I guess that at a jenkins level, it should be already in the environment, as this was working before the import was added.

Reply:
OK very well.